src/localization/localization.ts

```diff
@@ -248,4 +250,23 @@ function repository(type: RepositoryType): string {
     }
 }
 
+function dateSlot(slot: DateSlot) {
+    switch (slot) {
+        case DateSlot.Today:
+            return "Today";
+        case DateSlot.Yesterday:
+            return "Yesterday";
+        case DateSlot.LessThanAWeekAgo:
+            return "Less Than A Week Ago";
```

Comment:
Would it be better to make only the first letter capitalized? I'm not really used to this.

Reply:
You are right. Let me fix this.